### PR TITLE
Improved Hindi Language assets in languages/hindi.py

### PR DESCRIPTION
**Pull request by Sahethi**
Hey, I worked on this issue referring [https://phabricator.wikimedia.org/T223383](https://phabricator.wikimedia.org/T223383) but for the Hindi language. 
The comment under this issue mentioned that any other language would also be accepted. 
Added 13 informal words and 13 bad words.

### CODE REVIEW

**Comment by halfak** [comment]
@Sharingan-Coder01, can you take a look at this?

File: tests/languages/test_hindi.py [hunk 1]
```diff
@@ -62,6 +64,18 @@
...
     "हरामखोर",
+    "बहनचोद",
+    "गांडू",
+    "सूअर",
+    "हरामी",
+    "हराम जादा",
+    "लंड",
+    "कुतिया",
+    "कमीना",
+    "कमिनी",
```
Comment (Sharingan-Coder01) [inline tests/languages/test_hindi.py:73]: This spelling looks wrong to me. It should be " कमीनी "

Comment (Sahethi) [reply tests/languages/test_hindi.py:73]: @Sharingan-Coder01 okay, I'll make the required changes

**Comment by Sharingan-Coder01** [comment]
> @Sharingan-Coder01, can you take a look at this?

Looks good to me @halfak . Thanks for the help @Sahethi

**Comment by Sahethi** [comment]
@Sharingan-Coder01 great, thank you!

**Review by halfak — APPROVED** [review]
This all looks great to me, but someone from the ML team will need to do the merge.

**Comment by halfak** [comment]
Nevermind!  Looks like I do have rights.  Merging.  Thanks @Sharingan-Coder01 and @Sahethi!

**Comment by Sahethi** [comment]
@halfak Ohh great! Thank you!

**Pull request merged**